packages/peregrine/lib/talons/RootComponents/Category/categoryContent.gql.js

```diff
@@ -31,7 +31,23 @@ export const GET_CATEGORY_CONTENT = gql`
     }
 `;
 
+export const GET_CATEGORY_AVAILABLE_SORT_METHODS = gql`
+    query getCategoryAvailableSortMethods(
+        $categoryIdFilter: FilterEqualTypeInput!
+    ) {
+        products(filter: { category_id: $categoryIdFilter }) {
```

Comment:
We are using category uid now, 
```suggestion
        products(filter: { category_uid: $categoryIdFilter }) {
```